src/components/Drawer/Drawer.tsx

```diff
@@ -176,7 +192,7 @@ class Drawer extends React.PureComponent<Props, never> {
   }
 
   renderLayer() {
-    const { active, mode, componentLabel, componentWidth, theme } = this.props;
+    const { active, mode, componentLabel, componentWidth, theme,  } = this.props;
     const containerClassName = this.getContainerClassName();
```

Comment:
Is the extra comma intentional ?

Reply:
@ankitsaraogi its in tsconfig that every object should end with comma, that's why its there